ibis/omniscidb/tests/conftest.py

```diff
@@ -200,3 +212,77 @@ def temp_database(con, test_data_db: str) -> str:
     finally:
         con.set_database(test_data_db)
         con.drop_database(name, force=True)
+
+
+@pytest.fixture
+def temp_view(con) -> str:
+    """Return a temporary view name.
+
+    Parameters
+    ----------
+    con : ibis.omniscidb.OmniSciDBClient
+
+    Yields
+    ------
+    name : string
+        Random view name for a temporary usage.
+    """
+    name = _random_identifier('view')
+    try:
```

Comment:
In `temp_table` you're dropping the table before returning the name, but not here for the view. Any reason for the inconsitency?

Reply:
fixed